metrics/psutils/psutils_linux.lua

```diff
@@ -0,0 +1,69 @@
+local fio = require('fio')
+local string = require('string')
+local ffi = require('ffi')
+
+ffi.cdef[[
+    int get_nprocs_conf(void);
+]]
+
+local stat_file_path = os.getenv('TEST_STAT_FILE_PATH') or '/proc/stat'
```

Comment:
this should be refactored, it is pretty dangerous to use test only switches in production builds

Reply:
I refactored the test

Reply:
@oleggator 
> I refactored the test

Does it mean that #72 could be closed?